my.css

```diff
@@ -0,0 +1,280 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header .header {
+  background: black;
+  position: relative;
+}
+
+.logo {
+  position: absolute;
+  width: 98px;
+  height: 37px;
+  left: 52px;
+  top: 36px;
+  color: white;
+  font-family: Arial, sans-serif, Montserrat;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 30px;
+  line-height: 37px;
+}
+
+.ru {
+  display: flex;
+  justify-content: space-around;
+  position: absolute;
+  width: 31px;
+  height: 25px;
+  left: 388px;
+  top: 42px;
+  font-family: Arial, sans-serif, Montserrat;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.ru__link {
+  margin-left: 25px;
+  width: 11px;
+  height: 5.5px;
+  text-decoration: none;
+  color: #fff;
+}
+
+.burgermenu {
+  position: absolute;
+  left: 1163px;
+  top: 0;
+  width: 60px;
+  height: 60px;
+  background-color: #dcdcdc;
+}
+
+.burger {
+  width: 28px;
+  height: 5px;
+  border-radius: 40%;
+  margin: 22px 0 0 15px;
+  background-color: #000;
+  display: block;
+}
+
+.burger::before {
+  content: "";
+  position: absolute;
+  width: 28px;
+  height: 5px;
+  border-radius: 40%;
+  margin: 10px 0 0 0;
+  background-color: #000;
+  display: block;
+}
+
+/* первая секция */
+
+.collection {
+  display: flex;
+  height: 900px;
+  background:
+    linear-gradient(
+          90deg,
+          rgb(0, 0, 0) 39%,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)